getstarted/guides/index.markdown

```diff
@@ -47,20 +47,23 @@ Designed to get you started with enterprise mobile development.
 
 > Secure the application by using authentication, two-factor authentication and authorization mechanisms
 
-* [Two-Factor Authentication with One-Time Password - Android and iOS](/docs/guides/AeroGear-OTP)
+#### 1. Overview
 
-<h4 id="securityAndroid">Android</h4>
-* [Authentication](/docs/guides/aerogear-android/auth)
-* [Authorization](/docs/guides/aerogear-android/authz)
+In progress
 
-<h4 id="securityIOS2">iOS</h4>
-* [Authorization](/docs/guides/aerogear-ios-2.X/Authorization)
+#### 2. OTP
+* [AeroGear Two-Factor Authentication with OTP](AeroGear-OTP)
+* [AeroGear Two-Factor Authentication with OTP as Cordova Plugin](aerogear-cordova/AerogearCordovaOTP)
 
-<div class="alert alert-info" role="alert">If you are looking for AeroGear iOS 1.x series, please visit <a href="/docs/guides/aerogear-ios/">this page</a> for more information.</div>
```

Comment:
feel free to remove this '1.x series note' is already mentioned in up in the page and doesn't add any value